pkg/constants/constants.go

```diff
@@ -131,8 +133,9 @@ var (
 
 // Controller Constants
 var (
-	ControllerLabelName = KServeName + "-controller-manager"
-	DefaultMinReplicas  = 1
+	ControllerLabelName          = KServeName + "-controller-manager"
+	DefaultMinReplicas           = 1
+	IstioSidecarUIDAnnotationKey = KServeAPIGroupName + "/storage-initializer-uid"
```

Comment:
Are we planning to make this configurable [here](https://github.com/kserve/kserve/blob/master/config/configmap/inferenceservice.yaml#L52) also without having to add this annotation to every inference service?

Reply:
The "issue" is, that this is specific to the namespace where the service is deployed. At the moment there is no better way than to specify it per InferenceService. We're working with the istio folks to hopefully get rid of the underlying issue.